In [12]:

# Loading Libraries
import pandas as pd
import matplotlib.pyplot as plt
%matplotlib inline
import seaborn as sns
import plotly_express as px

ModuleNotFoundError: No module named 'plotly_express'

In [13]:
df=pd.read_csv("user_works.csv") #Loading Data
df.shape

(118471, 4)

In [14]:
df.head() 

user_id  company_id  industry_id  over_1000_employees
0        0           1            2                  0.0
1      151           1            2                  0.0
2      151           1            9                  0.0
3       41           1            2                  0.0
4       41           1            9                  0.0

In [15]:
df=df.drop(['over_1000_employees'],axis=1) # droping entire column 


In [16]:
df.head()

user_id  company_id  industry_id
0        0           1            2
1      151           1            2
2      151           1            9
3       41           1            2
4       41           1            9

In [17]:
# finding datatypes
df.dtypes

user_id        int64
company_id     int64
industry_id    int64
dtype: object

In [11]:
px.box(df, x='company_id',
       orientation='h', 
       hover_data=['company_id',])

NameError: name 'px' is not defined

In [10]:
df.count()

user_id        118471
company_id     118471
industry_id    118471
dtype: int64

# Insights
<ol>
    <li>Total 118471 company_id user </li>
    <li>Drop over_1000_employees due to more then 60% missing values</li>
    